torch/csrc/api/include/torch/nn/options/pooling.h

```diff
@@ -144,5 +144,28 @@ using MaxUnpool2dOptions = MaxUnpoolOptions<2>;
 /// `MaxUnpoolOptions` specialized for 3-D maxunpool.
 using MaxUnpool3dOptions = MaxUnpoolOptions<3>;
 
+// ============================================================================
+
+/// Options for a `D`-dimensional lppool functional and module.
+template <size_t D>
+struct LPPoolOptions {
+  LPPoolOptions(ExpandingArray<D> kernel_size)
+      : kernel_size_(kernel_size), stride_(kernel_size) {}
+
+  TORCH_ARG(float, norm_type) = 1.0f;
```

Comment:
I believe `norm_type` doesn't have a default value in Python version

Reply:
I removed default value.